java-action-archetype/.travis/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
Shouldn't this be executable ?
`-rw-r--r--  1 root  root   118 Feb 22 07:16 .//java-action-archetype/.travis/build.sh`

Reply:
Afterwards please include `java-actions-archetype` in https://github.com/apache/incubator-openwhisk-devtools/blob/master/.travis.yml#L7 so that Travis picks it up.